detection_rules/main.py

```diff
@@ -50,7 +50,7 @@ def root(ctx, debug):
               help='Type of rule to create')
 def create_rule(path, config, required_only, rule_type):
     """Create a detection rule."""
-    contents = load_rule_contents(config, single_only=True)[0] if config else {}
+    contents = load_rule_contents(Path(config), single_only=True)[0] if config else {}
```

Comment:
Just convert the type on L47 to `Path` and that should do it

Reply:
TBH i like the file exist checks that click.Path provides.